package.json

```diff
@@ -50,7 +50,8 @@
     "bluebird": "^3.5.1",
     "uuid": "^3.1.0",
     "eventemitter3": "^3.0.0",
-    "better-curry": "^1.6.0"
+    "better-curry": "^1.6.0",
+    "snyk": "^1.192.3"
```

Comment:
This should probably be a dev-dependency, right? Otherwise anyone using this library will also have to install snaky... 🤔

Reply:
indeed, but since the master is going through a major rewrite, it shouldn't matter ATM (and will fix this when my local changes are commited)